frontend/src/components/workspaces/WorkspaceWelcome/WorkspaceWelcome.tsx

```diff
@@ -0,0 +1,29 @@
+import logo from '../logo.svg';
+import Box from '../common/Box';
+import { FC } from 'react';
+
+export interface IWorkspaceWelcomeProps {}
+
+const WorkspaceWelcome: FC<IWorkspaceWelcomeProps> = ({ ...args }) => {
+  return (
+    <Box headTitle={'Welcome to CrownLabs'}>
+      <div className="w-full flex-grow flex flex-wrap content-center justify-center py-5 2xl:py-52">
+        <div className="w-full pb-10 flex justify-center">
+          <img
+            width={150}
+            src={logo}
+            style={{ color: '#1890ff' }}
+            alt="Logo Crownlabs"
+          />
+        </div>
+        <p className="text-3xl text-center px-24">
```

Comment:
24 padding is too much on mobile

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)